### DISCUSSION TITLE
Selection color should be a preference, not a style

### DISCUSSION THREAD

**Discussion by burnuser** [freeplane/freeplane#430] [Ideas]
Tools => Preferences => Defaults: Selection colors => Standard selected node color is overruled by Styles - but it should not.

1.) The background color of a selected node is very important for the readability of the node-text.
A relatively dark blue like #4e85f8 behind black Text  is a very bad choice in this context.
A lighter color like the traditional #ccffff is much better for the easy distinction of text and background.
(And in earlier Freeplane versions - e.g. 1.6.15 - the problem does not exist.)

2.) The background color of a selected node is not a real format like the permanent background color of a node. Instead it is a general visual helper for editing and should be adjusted in preferences, not in styles. The only style context is the use of dark styles with light Text colors, which needs an inverted background color. (But e.g. in Darcula, both above standard colors are to light.)

3.) I was not able to change the standard background color of selected nodes for new maps anyway. The setting in Preferences is ignored for e.g. "standard 1.6 noEdgeColor", and the style setting for background color of a selected node works only in current map.

I like Freeplane, but the stated color problem is very anoying.

**Comment by macmarrum** [comment]
Until `Selected node` was introduced as a style defined on the map level, the background color was a Freeplane-wide setting, similar to today's color of the bubble when `Defaults->Selection colors->Display selected nodes in bubbles` is on.

The functionality of a `Selected node` style gives us the opportunity for each mind map to have a dedicated (color) theme covering also how selected nodes should be formatted. This makes sense, because dark maps will probably need a bright "Selected node", and vice versa.

There is an option in Preferences `Defaults->Selection colors->Standard selected node color (fallback)`. This will work as a Freeplane-wide setting, but only as a fallback. The option's tooltip says "The fallback is used if the color is not set in the map styles". This is misleading. From what I can tell, the color is used only when `Selected style` is missing in the mind map. The thing is that when I try to delete `Selected node`, Freeplane tells me "Can not delete predefined style". The only way I managed to delete it, was to open the .mm file in a text editor, find the xml node `stylenode LOCALIZED_TEXT="defaultstyle.selection"` and deleted it there. 
```xml
<stylenode LOCALIZED_TEXT="defaultstyle.selection" BORDER_COLOR_LIKE_EDGE="false" BORDER_COLOR="#ff6600"/>
</stylenode>
```
In other words, the option is there to handle mind maps created with earlier versions of Freeplane, where `Selected node` style was still missing.

@burnuser you might want to [create a feature request](https://github.com/freeplane/freeplane/issues/new/choose) to add another option e.g. `Ignore map-level Selected node` (as if it didn't exist), which would force Freeplane to use `Defaults->Selection colors->Standard selected node color (fallback)` instead

**Reply by burnuser** [reply]
Thank You macmarrum!

Yes, I also found the **fallback** hint, but in the German translation (originally used by me) it is missing. So in German version, the option is definitely misleading.

Anyway, following your suggestion, I will create a feature request!
